tasks/main.yml

```diff
@@ -20,7 +20,7 @@
   tags:
     - haproxy
 
-- name: Start Marathon service
-  service: name=marathon state=restarted enabled=yes
+- name: Enable Marathon service
+  service: name=marathon enabled=yes
```

Comment:
> as the handler will ensure the service is started in a ll cases

currently it will start Marathon on next boot. Please add ```state=started``` no need for any extra variables